.circleci/config.yml

```diff
@@ -369,12 +363,9 @@ workflows:
       - deprecated_code:
           requires:
             - build
-      - unit_kernel_tests:
-          requires:
-            - build
 
-  # A nightly build of the project, using all dof-dss packages at HEAD from development branch.
-  nightly-edge-build:
+  # A daily build of the project, using all dof-dss packages at HEAD from development branch.
```

Comment:
Left daily in the comment, which matches the cron syntax. No reason why it should be nightly... it might execute in sunlit hours at some point in the future... who knows!